data/json/recipes/recipe_others.json

```diff
@@ -7758,6 +7758,23 @@
   ]
 },{
   "type" : "recipe",
+  "result": "washboard",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_TOOLS",
+  "skill_used": "fabrication",
+  "difficulty": 1,
+  "time": 1000,
+  "autolearn": true,
+  "qualities" : [
+    {"id":"CUT","level":1}
+  ],
+  "components": [
+    [
+      [ "2x4", 1]
+    ]
+  ] 
```

Comment:
Formatting: it would look better as one line rather than 5. Same for qualities above.